.github/workflows/push.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php_version: [ '7.2', '7.3', '7.4', '8.0', '8.1' ]
+        php_version: [ '7.2', '7.3', '7.4', '8.0', '8.1', '8.2', '8.3', '8.4' ]
```

Comment:
We're about to upgrade to `mediawiki-codesniffer@45`, which requires php >= 7.4. I guess it makes sense to drop the earlier versions already.

Reply:
Great - thank you!